NFC-reader/README.md

```
@@ -35,32 +37,131 @@ The adapted command mode, enables the system to input a stream of pixel from the
+## X-NUCLEO-NFC07A1
+
+As an alternative, I decided to use a simpler device, **X-NUCLEO-NFC07A1**, which serves as a **dynamic NFC tag**. Communication with this device is much simpler, thanks to the availability of a **BSP component**. 
 
-#### List of BSP components
+The device communicates via **I2C** and allows storing various types of information. In my case, I chose to store a link to a web page.
 
+## BSP components
+
+My project includes individual **BSP components** for device communication.
+
+#### List
```

Markdownlint (0.37.0)

Expected: h3; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)